near/nearcore
# rename debug pages: chain -> sync, sync -> network

### Description
This PR renames & restructures a few pages under Debug. The "chain info" page is merged with the "chunk info" page, and together they are now called "Chain & Chunk info", and the "sync info" page is now called "network info". The purpose is to group the page contents as close as possible and reduce the effort need to locate debug information.

### JIRA Issue
[CP-82](https://nearinc.atlassian.net/browse/CP-82)

### Test plan

 - Build with make neard
 - Under `nearcore` directory, launch localnet using command `nearup stop && nearup run localnet --binary-path ./target/release`
 - Open your browser and go to http://[host_address]:3030/debug/
 - Expect to see the link "Chain & Chunk Info" and "Network Info"
 - Click the link "Chain & Chunk Info"
 - Expect to see a webpage that looks similar to the one below
 - Click the link "Network Info"
 - Expect to see a webpage that looks similar to the one below

### Screenshot on testnet (includes changed pages only)
<img width="449" alt="Screen Shot 2022-05-10 at 4 22 52 PM" src="https://user-images.githubusercontent.com/98097537/167741669-07049822-5f72-4e84-a2ba-7c278c86015a.png">
<img width="1512" alt="Screen Shot 2022-05-10 at 4 23 16 PM" src="https://user-images.githubusercontent.com/98097537/167741674-b5cdb89d-24d3-487d-8fcc-e3c1c7ae2068.png">
<img width="1180" alt="Screen Shot 2022-05-10 at 4 23 47 PM" src="https://user-images.githubusercontent.com/98097537/167741680-913ed59e-a01c-4a6d-b084-c6a2410fd3a6.png">


### First round reviewer
 - @mzhangmzz 
 - @mm-near
